test_args.sh

```diff
@@ -67,11 +67,17 @@ run_lp -broadcaster -datadir "$CUSTOM_DATADIR"
 [ ! -d  "$CUSTOM_DATADIR"/offchain ] # sanity check that network isn't included
 kill $pid
 
+CUSTOM_DATADIR="$TMPDIR"/customDatadir2
+
 # check custom datadir with a network
 run_lp -broadcaster -datadir "$CUSTOM_DATADIR" -network rinkeby $ETH_ARGS
 [ ! -d  "$CUSTOM_DATADIR"/rinkeby ] # sanity check that network isn't included
 kill $pid
 
+# expect error because re-use network name with different chainID
+run_lp -broadcaster -datadir "$CUSTOM_DATADIR" -network mainnet $ETH_ARGS || res=$?
```

Comment:
I ran this locally and I think you may need to increase SLEEP to something like .5 or else the DB is not initialized in the previous command before the process is killed